src/WikiService.Models.Db/DbRubric.cs

```diff
@@ -12,6 +12,7 @@ public class DbRubric
     public Guid Id { get; set; }
     public string Name { get; set; }
     public Guid? ParentId { get; set; }
+    public int? ChildCount { get; set; }
```

Comment:
for what?

Reply:
Just to tell front end side are there any child rubrics or not